namer/namer.cc

```diff
@@ -1007,6 +1039,21 @@ class SymbolDefiner {
         }
     }
 
+    void modifyConstant(core::MutableContext ctx, const Modifier &mod) {
+        ENFORCE(mod.kind == Modifier::ClassOrStaticField);
+
+        auto owner = ctx.owner.data(ctx)->enclosingClass(ctx);
+        auto constantNameRef = ctx.state.lookupNameConstant(mod.target);
+        auto constant = ctx.state.lookupSymbol(owner, constantNameRef);
+        if (constant.exists() && mod.name._id == core::Names::privateConstant()._id) {
```

Comment:
```suggestion
        if (constant.exists() && mod.name == core::Names::privateConstant()) {
```

Reply:
technically `_id` is a "private" field but we don't actually make it private because otherwise there's unfortunately no way to `switch` over NameRef's. So we only use it in `switch` / `case` statements.